### ISSUE TITLE
[1.19.3]Fire Animation

### ISSUE THREAD

**Issue by gnilobaiter** [Asek3/Rubidium#390]
Hi, I ran into a problem while using Rubidium on version 1.19.3 (Forge - 44.1.0). Animations of fire and bonfires do not work (perhaps something else, I did not notice, smoke is coming from the campfire). I will attach a list of mods and a GIF below. Animations are all on in options

**https://i.imgur.com/RpX0TQs.mp4**

![image](https://user-images.githubusercontent.com/104412752/212475601-f3561958-dc6f-4795-868a-0f0beb1c5765.png)

**Comment by gnilobaiter** [comment]
![image](https://user-images.githubusercontent.com/104412752/212475878-73356f04-15d0-4d4c-91ba-4b78392292e9.png)
I want to add that if you turn off this option, it crashes when you enter any world

**Comment by clogosz** [comment]
Also seeing this issue, though with a much smaller mod list. That said, I also notably have MouseTweaks and Xaero's mods installed.

**Comment by gnilobaiter** [comment]
I tryied another version of forge, but this doesn't help :(

**Comment by acemagex** [comment]
Can confirm this issue, including crashing when toggling "animate only visible textures." Disabling the mod fixes the animations.

**Comment by ZilverLazarus** [comment]
Can also confirm. This just happened to me using Rubidium 0.6.3 on Minecraft 1.19.3 + Forge 44.1.5.

Here is my crash log:
[crash-2023-01-21_14.09.39-client.txt](https://github.com/Asek3/Rubidium/files/10472836/crash-2023-01-21_14.09.39-client.txt)

**Comment by ni15** [comment]
also happening to me
[Uploading latest.log…]()

**Comment by ProjectSky** [comment]
same issues

* crash-client.txt
```
org.spongepowered.asm.mixin.transformer.throwables.IllegalClassLoadError: Illegal classload request for accessor mixin me.jellysquid.mods.sodium.mixin.features.texture_updates.SpriteInfoAnimationAccessor. The mixin is missing from rubidium.mixins.json which owns package me.jellysquid.mods.sodium.mixin.* and the mixin has not been applied.
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:334) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.8.jar:10.0.8+10.0.8+main.0ef7e830] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.6.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
	at net.minecraft.client.renderer.texture.SpriteContents$InterpolationData.m_245152_(SpriteContents.java:540) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:features.texture_updates.MixinSpriteInterpolated,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.texture.SpriteContents$Ticker.m_247697_(SpriteContents.java:346) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:features.texture_tracking.MixinSpriteContentsAnimatorImpl,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.texture.TextureAtlasSprite$1.m_245385_(TextureAtlasSprite.java:57) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:classloading}
	at net.minecraft.client.renderer.texture.TextureAtlas.m_118270_(TextureAtlas.java:113) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:rubidium.mixins.json:features.texture_tracking.MixinSpriteAtlasTexture,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.texture.TextureAtlas.m_7673_(TextureAtlas.java:122) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:rubidium.mixins.json:features.texture_tracking.MixinSpriteAtlasTexture,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.texture.TextureManager.m_7673_(TextureManager.java:161) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1758) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1097) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:713) ~[client-1.19.3-20221207.122022-srg.jar%23186!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.m_239872_(Main.java:212) ~[1.19.3-Forge_44.1.16.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:51) ~[1.19.3-Forge_44.1.16.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.3-44.1.16.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:58) ~[?:?] {}
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?] {}
```

**Comment by Asek3** [comment]
Fixed on other versions, 1.19.3 is abandoned
